packages/spotlight/src/target.js

```diff
@@ -73,9 +72,9 @@ function getTargetByContainer (containerId) {
 function getTargetBySelector (selector) {
 	if (!selector) return null;
 
-	if (selector.charAt(0) === '@') {
-		const containerId = selector.length === 1 ? null : selector.substr(1);
-
+	if (selector[0] === '@') {
+		//  no need to check selector.length === 1 and assign null.
```

Comment:
It does seem like an empty string and `null` would result in the same outcome.

Reply:
same outcome?

Reply:
I just meant that you are right that passing an empty string down through the methods does result in the same as passing `null`, but I had to check through the other functions called here.